Mathlib/MeasureTheory/Function/LpSpace/Basic.lean

```diff
@@ -688,21 +688,22 @@ end LipschitzWith
 namespace ContinuousLinearMap
 
 variable {𝕜 : Type*} [NontriviallyNormedField 𝕜] [NormedSpace 𝕜 E] [NormedSpace 𝕜 F]
+variable {σ : 𝕜 →+* 𝕜} [RingHomIsometric σ]
```

Comment:
Like in the other PR, could you generalize this to `variable {σ : 𝕜 →+* 𝕜'} [RingHomIsometric σ]` where 
`[NontriviallyNormedField 𝕜'] [NormedSpace 𝕜' F]`?